rb/build.desc

```diff
@@ -146,6 +146,9 @@ ruby_library(name = "ie",
     "lib/selenium/webdriver/ie/**/*.rb",
     "lib/selenium/webdriver/ie.rb"
   ],
+  resources = [
+    { "//javascript/webdriver/atoms:getAttribute": "rb/lib/selenium/webdriver/atoms/getAttribute.js"}
```

Comment:
Since we now have the protocol handshake, we should probably add this to common instead of including in each one. The only one that shouldn't need it in the long run would be legacy FF and it's not that much overhead